src/main/java/com/aua/flightreservationsystem/core/admin/Admin.java

```diff
@@ -0,0 +1,15 @@
+package com.aua.flightreservationsystem.core.admin;
+
+import com.aua.flightreservationsystem.persistence.model.RoleInCompany;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.UUID;
+
+@Value
+@Builder(toBuilder = true)
+public class Admin {
+    UUID id;
+    RoleInCompany roleInCompany;
+    UUID userId;
```

Comment:
check the admin Entity variable name , and type it correctly for all the other modals as well

Eg.
```suggestion
    User user;
```